core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1926,4 +1927,9 @@ CacheConfigurationException offHeapMemoryEvictionSizeNotLargeEnoughForAddresses(
 
    @Message(value = "Thread Pool Factory %s is blocking, but this pool requires non blocking threads", id = 566)
    CacheConfigurationException threadPoolFactoryIsBlocking(String name);
+
+   @LogMessage(level = WARN)
+   @Message(value = "Unable to persist Infinispan internal caches as no global state enabled")
```

Comment:
I think it would be worth printing the name of the internal cache. Why print it just ```@Once```? This WARN should be as annoying as possible!

Reply:
I added `@Once` as I don't really see any value in creating a log for all of the internal caches that are affected. IMO unless we can't avoid it, the name of internal caches shouldn't be exposed to the user, they just need to be aware that global state is required in order for them to function as intended.